x-pack/plugins/canvas/public/lib/aeroelastic/config.js

```diff
@@ -8,12 +8,16 @@
  * Mock config
  */
 
+const adHocGroupName = 'adHocGroup';
 const alignmentGuideName = 'alignmentGuide';
 const atopZ = 1000;
 const depthSelect = true;
 const devColor = 'magenta';
+const groupName = 'group';
```

Comment:
is the idea that this gets used for persistent grouping later on?

Reply:
@rashidkpc yes, correct! The `adHocGroup` is merely a sub-type.